company-profiles/carbon-black.md

```diff
@@ -1,11 +1,85 @@
-# Carbon Black
+# Bairesdev
```

Comment:
Can you change this, please?

Reply:
updated, sorry about that ✌️

Reply:
Looks like you updated the wrong file there.

Reply:
Can this one be changed back to Carbon Black, please?